### PR TITLE
dry-validations bump to 1.0

### PR DESCRIPTION
**Pull request by mensfeld**
close https://github.com/karafka/karafka/issues/483

### CODE REVIEW

File: lib/karafka/schemas/consumer_group.rb [hunk 1]
```diff
@@ -3,7 +3,9 @@
 module Karafka
   module Schemas
     # Schema for single full route (consumer group + topics) validation.
-    ConsumerGroup = Dry::Validation.Schema do
+    class ConsumerGroup < Dry::Validation::Contract
+      config.messages.load_paths << File.join(Karafka .gem_root, 'config', 'errors.yml')
```
Comment (tappleby) [inline lib/karafka/schemas/consumer_group.rb]: Is there supposed to be a space here `Karafka .gem_root`? Noticed ResponderUsageTopic also has the same syntax.

Comment (mensfeld) [reply lib/karafka/schemas/consumer_group.rb]: Good catch. Checking why the CI didn't catch that...

Comment (mensfeld) [reply lib/karafka/schemas/consumer_group.rb]: Dunno. Not catched. Fixed. Thanks!

Comment (tappleby) [reply lib/karafka/schemas/consumer_group.rb]: Apparently it’s actually valid syntax 🤷‍♂️

Comment (mensfeld) [reply lib/karafka/schemas/consumer_group.rb]: Yeah. Weird. I believe RuboCop should catch that and Coditsu should fail on Circle. Didn't but fixed. Thank you :heart:

**Pull request merged**